apps/old/widget/components/Button.jsx

```diff
@@ -47,6 +49,8 @@ const StyledButton = styled.button`
         return "var(--button-primary-color, #000)";
       case "outline":
         return "var(--button-outline-color, #fff)";
+      case "danger":
```

Comment:
Along with adding the "danger" variant, you can include the change that originally motivated the variant:

There exists a red "Delete Project" button on the project editor page, could you modify to now use this variant please?